src/components/details.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Card from 'react-bootstrap/Card';
+import { useLocation } from 'react-router-dom';
+
+const Details = () => {
```

Comment:
- [ ] In your detail page, kindly ensure you follow the design template. In the original design, there is a banner section, that contains **CZECH REPUBLIC 954 VIEWS** and then the other section containing cities in the Czech Republic. In your own implementation, there is only one section - the rest of the page is blank. Kindly separate the banner section (for example Espirito Santo) from the other details (Total cases confirmed, Cases confirmed today, Deaths confirmed today, Information source) so that they follow the design template.

- [ ] Kindly ensure this change reflects in your live demo as well.

  Design Template | Your Implementation
  :---:|:---:
  ![jonathas-origin-detail](https://user-images.githubusercontent.com/36542986/152303229-d8b5fda5-f676-48a3-87fc-6afa6a0c1781.jpg) | ![jonathas-own-detail](https://user-images.githubusercontent.com/36542986/152303244-bfff6a71-87d9-4208-885f-68816541075f.jpg)